dist/cookie/{{cookiecutter.project_slug}}/android/fastlane/Fastfile

```diff
@@ -10,121 +10,12 @@
-                             version_code: ENV["CIRCLE_BUILD_NUM"].to_i,
-                             version_name: ENV["CIRCLE_BUILD_NUM"].to_s,
-                             track_promote_to: 'alpha',
-                             skip_upload_metadata: true)
-      end
-    end
-  end
-
-  desc('Promote from internal to alpha')
-  lane(:alpha) do
-    upload_to_play_store(rollout: '1', track: 'internal', track_promote_to: 'alpha')
-  end
-
-  desc('Promote from internal to beta')
-  lane(:beta) do
-    versions = google_play_track_version_codes(track: "internal")
-    version_code = versions[0]
-
-    upload_to_play_store(rollout: '1', track: 'internal', track_promote_to: 'beta', version_code: version_code, skip_upload_metadata: true)
-  end
-
-  desc('Deploy a new version to the Google Play')
-  lane(:production) do
-    versions = google_play_track_version_codes(track: "beta")
-    version_code = versions[0]
-
-    upload_to_play_store(rollout: '1', track: 'beta', track_promote_to: 'production', version_code: version_code, skip_upload_metadata: true)
-  end
-
-  desc('Deployment to Appetize')
-  lane(:deploy_appetize) do
-    gradle(task: 'assemble', build_type: 'Release')
-
-    appetize(
-      path: get_last_apk_path,
-      platform: 'android',
-      note: base_name
-    )
-
-    update_url(
-      platform: 'android',
-      public_key: lane_context[SharedValues::APPETIZE_PUBLIC_KEY],
-      url: lane_context[SharedValues::APPETIZE_APP_URL]
-    )
-  end
-
-  desc('Create a new Android app bundle build')
-  lane(:build_aab) do
-    build_version = increment_version_code(
-      version_code: ENV["CIRCLE_BUILD_NUM"].to_i
-    )
-
-    gradle(task: 'bundleRelease')
-    sh "echo \"export BUILD_VERSION=#{build_version}\" >> $BASH_ENV"
-    sh ". $BASH_ENV"
-  end
-
-  # Update app URL in CB app DB
-  private_lane(:update_url) do |options|
-    url = "https://#{ENV['WEBHOOK_HOSTNAME']}/api/v2/apps/#{ENV['PROJECT_ID']}/metadata-webhook/"
-    data = {
-      platform_id: ENV['PLATFORM_ID'],
-      provider: 'circleci',
-      metadata: {
-        appetize: {
-          options[:platform] => {
-            version: ENV['CIRCLE_BUILD_NUM'],
-            public_key: options[:public_key],
-            url: options[:url]
-          }
-        }
-      }
-    }
-    headers = {
-      'Authorization': "Api-Key #{ENV['WEBHOOK_API_KEY']}",
-      'Content-Type': 'application/json'
-    }
-
-    response = HTTParty.post(url, body: data.to_json, headers: headers)
-    puts("API response: #{response.code} #{response.body}")
-
-    response.success?
-  end
-end
-
-def get_last_apk_path
-  last_apk_path = lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH]
-  puts("last_apk_path: #{last_apk_path}")
-
-  last_apk_path
-end
+import_from_git(
+  url: "git@github.com:crowdbotics/fastlane-config.git", # The URL of the repository to import the Fastfile from.
+  branch: "master", # The branch to checkout on the repository.
```

Comment:
I checked on the https://github.com/crowdbotics/fastlane-config repo and the default branch is set to `main` not `master`. I would say it's better to change the default branch on that repo so you don't make more changes here and because I believe most of our repos use `master` so we should stick to it.

Reply:
I'll make the change!